examples/file_parser.rs

```diff
@@ -8,8 +8,8 @@ extern crate webrtc_sdp;
 fn main() {
     let filename = match env::args().nth(1) {
         None => {
-            println!("Missing file name argument!");
-            return;
+            eprintln!("Missing file name argument!");
+            std::process::exit(1);
```

Comment:
Great, is this being run in CI currently?

Reply:
Do you mean if the file_parser gets invoked/run in CI at all?
The answer would be no.